modules/end-user-guide/pages/git-credential-store.adoc

```diff
@@ -1,25 +1,48 @@
-:navtitle: Git credential store
+:navtitle: Creating a {orch-name} Secret for a Git credentials store
 :keywords: user-guide, configuring, user, secrets
 :page-aliases: 
 
-[id="mounting-a-git-credential-store-into-a-workspace-container_{context}"]
-= Mounting a Git credentials store into a workspace container
+[id="mounting-a-git-credential-store-into-workspace-containers_{context}"]
+= Creating a {orch-name} Secret for a Git credentials store
 
-This section describes how to mount Git credentials store as secret from the user's {orch-namespace} into the file in single-workspace or multiple-workspace containers of {prod-short}.
+This section describes how to create a Secret containing a Git credentials store in the user's {orch-namespace}. The Git credentials store specified in the Secret is mounted as a file in the {devworkspace}'s containers.
 
 .Prerequisites
 
 * A running instance of {prod-short}.
+* You have CLI or GUI access to the {platforms-name} cluster of your organization's {prod-short} instance.
 
 .Procedure
-. Prepare Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
-. Encode content of the file to the base64 format.
-. Create a new {platforms-name} secret in the {platforms-name} {orch-namespace} where a {prod-short} workspace will be created.
+. Prepare a Git credential file in the link:https://git-scm.com/docs/git-credential-store#_storage_format[Storage format].
 
-* The labels of the secret that is about to be created must match the set of labels configured in `che.workspace.provision.secret.labels` property of {prod-short}. The default labels are:
+. Encode content of the file to the base64 format.
```

Comment:
```suggestion
. Encode the selected content of the `.git-credentials` file to Base64 for the next step.
+
[TIP]
====
Linux CLI users can convert it to Base64 standard output by using `base64` as needed:

* Convert all lines in the file: `cat .git-credentials | base64`

* Convert a selected line: `$ echo -n '__<copied_and_pasted_line_from_.git-credentials>__' | base64`
====
```

Reply:
I think this is a bit wordy?

How about `CLI users can create a Base64-encoded string by running `base64`:`?

Reply:
@dkwon17, I initially included `as needed` to clarify a bit that the two bullet points are not two consecutive steps but are rather mutually exclusive alternatives. Up to you whether to keep `as needed` or drop it.

Reply:
I'm not sure if `base64` is available to non-Linux CLI users.

Reply:
Updated in my latest push

Reply:
<img width="544" alt="image" src="https://user-images.githubusercontent.com/83611742/163254255-d2da37a8-ecd0-462c-ba65-1a3d33525916.png">

Reply:
Updated again to mention standard output
<img width="550" alt="image" src="https://user-images.githubusercontent.com/83611742/163278196-91899b44-7a97-4b94-a746-043c405b06d2.png">